model/network/registry.yaml

```diff
@@ -236,3 +236,46 @@ groups:
         type: string
         brief: 'The network interface name.'
         examples: [ 'lo', 'eth0' ]
+      - id: network.connection.state
+        type:
+          members:
+            - id: close
+              value: 'close'
+              stability: experimental
+            - id: close_wait
+              value: 'close_wait'
+              stability: experimental
+            - id: closing
+              value: 'closing'
+              stability: experimental
+            - id: delete
+              value: 'delete'
+              stability: experimental
+            - id: established
+              value: 'established'
+              stability: experimental
+            - id: fin_wait_1
+              value: 'fin_wait_1'
+              stability: experimental
+            - id: fin_wait_2
+              value: 'fin_wait_2'
+              stability: experimental
+            - id: last_ack
+              value: 'last_ack'
+              stability: experimental
+            - id: listen
+              value: 'listen'
+              stability: experimental
+            - id: syn_recv
+              value: 'syn_recv'
+              stability: experimental
+            - id: syn_sent
+              value: 'syn_sent'
+              stability: experimental
+            - id: time_wait
+              value: 'time_wait'
+              stability: experimental
+        stability: experimental
+        brief: "The state of network connection"
+        note: "A stateless protocol MUST NOT set this attribute"
```

Comment:
I suggest to remove this note. Telemetry describing stateless protocols won't use this attribute and it seems obvious that stateless thing doesn't need a state.

Reply:
Makes sense to me to remove it

Reply:
Removed.